src/middleware/simplified-account/simplified-account-strategy.middleware.js

```diff
@@ -71,3 +71,22 @@ module.exports = async function getSimplifiedAccount (req, res, next) {
     next(err)
   }
 }
+
+/**
+ * An Express Request object extended with service, account, and user
+ * @typedef {Object} SimplifiedAccountRequest
```

Comment:
Could put a TODO here so we know to rename this when the old settings are deprecated.